internal/common/address.go

```diff
@@ -49,6 +50,12 @@ func IsBech32Address(s string) bool {
 	return true
 }
 
+// IsEmpty gets whether the address contains all 0 bytes
+func (a Address) IsEmpty() bool {
+	empty := Address{}
```

Comment:
pull out as plain var, no need to make each time.

Reply:
Done.